### PR TITLE
Add test suite for proxy auth header

### PR DESCRIPTION
**Pull request by atward**
Adds tests for #250 

Disclosure: I currently don't have vagrant. Tests are based of existing kitchen-ec2 testing I'm doing elsewhere.

### CODE REVIEW

**Comment by xorima** [comment]
Looks good, you could also look into dokken for tests if you are on linux or mac

**Review by xorima — DISMISSED** [review]

**Comment by xorima** [comment]
Can you resolve the conflicts please @atward

**Comment by atward** [comment]
Rebased
Tests are probably going to fail until #257 resolved.

**Review by xorima — DISMISSED** [review]

File: test/fixtures/cookbooks/test/recipes/proxy.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+grafana_config_auth 'Grafana' do
+  disable_login_form true
+  proxy_enabled true
+  proxy_header_name auth_header
+  proxy_header_property 'username'
+  proxy_whitelist '127.0.0.1, ::1'
+  login_cookie_name 'grafana_sess'
+end
+
```
Comment (xorima) [inline test/fixtures/cookbooks/test/recipes/proxy.rb:15]:
```suggestion
grafana_config_writer 'Grafana'
```

**Pull request merged**